scripts/gh-action-test.sh

```diff
@@ -0,0 +1,12 @@
+DOCKER_COMPOSE_VERSION=1.23.1
+
+sudo rm /usr/local/bin/docker-compose
+curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+chmod +x docker-compose
+sudo mv docker-compose /usr/local/bin
```

Comment:
Could you test this without lines 1-6.  I have a feeling this setup isn't necessary